requirements/requirements.in

```diff
@@ -11,10 +11,13 @@ django-filter
 django-axes
 
 # support
+backports.zoneinfo
 django-waffle
 django-storages
 python-dotenv
 jsonschema
+importlib-resources
+pkgutil-resolve-name
```

Comment:
These additions are actually just dependencies. We've always installed them, but it started complaining about the versions not being pinned in requirements.txt. I added them here explicitly to silence the error, but we've always installed these packages